packages/client/utils/makeCheckinGreeting.ts

```diff
@@ -54,15 +54,36 @@ const greetings = [
   {
     content: 'Zdravo',
     language: 'Slovenian'
+  },
+  {
+    content: 'Barev',
+    language: 'Armenian'
+  },
+  {
+    content: 'Bună',
+    language: 'Romanian'
+  },
+  {
+    content: 'Yasou',
```

Comment:
hey man, just cause we eat a lot of greek food don't mean we want it poppin up twice as much as all the other languages

Reply:
fixed!